src/orders/orders.cpp

```diff
@@ -18,12 +18,110 @@
 
 #include "orders.hpp"
 
+namespace antara
+{
+    pair pair::of (std::string a, std::string b)
+    {
+        antara::pair pair = {{st_symbol{a}}, {st_symbol{b}}};
```

Comment:
st_symbol{std::move(a)}, you take string by copy here, and you want to move the value inside the pair during the creation